nodes/usb_cam_node.cpp

```diff
@@ -362,15 +362,29 @@ class UsbCamNode
 
   void resetExposureSettings()
   {
+    // Set auto exposure to on
     cam_.is_changing_config(true);
     cam_.set_v4l_parameter(
       "auto_exposure",
       AUTO_EXPOSURE_APERTURE_PRIORITY_MODE
     );
+    cam_.is_changing_config(false);
```

Comment:
put this in the `set_v4l_parameter` function at beginning and end

Reply:
done